web/models/telemetry.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+import "time"
+
+type Telemetry struct {
```

Comment:
I have a super hard time with this name :smile: 

What is "a telemetry"? Wouldn't be more correct to denote this as `TelemetryPayload` or something like that?

Reply:
this is actually the database schema, telemetry payload suggests something that is not normalized, `HostTelemetry` or `HostStats` probably are a better fit here. wdyt?

Reply:
Yeah, me too :sweat_smile: 
See this https://github.com/trento-project/trento/pull/418#discussion_r741672789, most likely it would be a `HostTelemetry`

Reply:
Ouch, sorry for the duplicate comment then!

Reply:
fixed.